src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -138,26 +137,32 @@
             <tr>
                 <th>{{ show_level ? 'Level' : 'Time' }}</th>
                 {% if channel_is_defined %}<th>Channel</th>{% endif %}
-                <th>Message</th>
+                <th style="width: 100%">Message</th>
```

Comment:
I replaced this by `class="full-width"` recently (and added the corresponding class in profiler.css.twig